libs/live-wallet/src/store.ts

```diff
@@ -91,6 +115,24 @@ export const handlers: WalletHandlers = {
     }
     return { ...state, accountNames };
   },
+  WALLET_SYNC_UPDATE: (
+    state,
+    {
+      payload: {
+        newNamesState,
+        failures,
+        newState: { data, version },
+      },
+    },
+  ) => {
+    const wsStateNonImportedAccountIds = Object.keys(failures);
+    return {
+      ...state,
+      accountNames: newNamesState,
+      wsStateNonImportedAccountIds,
```

Comment:
problem 1: we override it over. we need to fusion it and remove those who were treated.
problem 2: in a generalisation, how would we address this state. do we really need to store this state ultimately, can we do without?